resource/ui/js/app.js

```diff
@@ -14,22 +14,23 @@
 // for names of contributors.
 //
 // Author: Andrew Bonventre (andybons@gmail.com)
-
+// Author: Bram Gruneir (bramgruneir@gmail.com)
+/// <reference path="typings/angularjs/angular.d.ts" />
+/// <reference path="typings/angularjs/angular-route.d.ts" />
 var crApp = angular.module('cockroach', [
```

Comment:
We should make it more clear that the .js files are not to be edited. Maybe put these files in ui/js/generated/?

Is it possible to get the typescript compiler to output a comment in the first line starting with "// Generated by "? That's what coffeescript does and github recognizes it to hide the output from github diffs.